tests/includeme_test.py

```diff
@@ -23,10 +23,20 @@ def test_bad_schema_validated_on_include():
         'pyramid_swagger.schema_directory': 'tests/sample_schemas/bad_app/',
     }
     mock_config = mock.Mock(registry=mock.Mock(settings=settings))
-    with pytest.raises(jsonschema.exceptions.ValidationError):
+    with pytest.raises(ValidationError):
         pyramid_swagger.includeme(mock_config)
 
```

Comment:
I'm not familiar enough with the context here, but I think it is good practice to attempt to assert the message and/or contents of the exception to tighten up the expectation.

```python
with pytest.raises(ValidationError) as excinfo:
    blah
assert 'failed blah blah blah' in str(excinfo.value)
```